Update README.md with possible projects (#43)
CliMA/CloudMicrophysics.jl: README.md

  (accessed by pressing `]` in the Julia command prompt):

```julia
julia>]
pkg> add CloudMicrophysics
pkg> instantiate
```

CloudMicrophysics.jl can be updated to the latest tagged release
  from the package manager.
The package is still under development and changes to API are very possible!

```julia
pkg> update CloudMicrophysics
```

When contributing to the CloudMicrophysics.jl development,
  the easiest way is to clone the [repository](https://github.com/CliMA/CloudMicrophysics.jl)
  and then run it using its project environment.
For example, to get all the needed dependencies and then run all the tests
  you could try:

```julia
julia --project

julia>]

pkg> instantiate

julia> include("test/runtests.jl")
```

## Contributing

The CloudMicrophysics.jl package is being actively developed
  and welcomes contributions and feedback.
There is a variety of projects big and small that are available to take up as
  fun research projects for interested students and other contributors.
Below is a list of possible examples,
  but other suggestions and ideas are always welcome!

- The CloudMicrophysics.jl package contains an aerosol activation parameterization

>   that is an implementation of [Abdul-Razzak and Ghan 2000](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/1999JD901161)

Reviewer: ```suggestion
  that is an implementation of [Abdul-Razzak and Ghan 2000](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/1999JD901161).
```

> - The CloudMicrophysics.jl package contains an aerosol activation parameterization
>   that is an implementation of [Abdul-Razzak and Ghan 2000](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/1999JD901161)

Reviewer 2: ```suggestion
- The CloudMicrophysics.jl package contains an aerosol activation parameterization
  that is an implementation of [Abdul-Razzak and Ghan 2000](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/1999JD901161).
```

  In the [documentation](https://clima.github.io/CloudMicrophysics.jl/dev/Aerosol_activation/#Example-figures)
  we show we can reproduce the first figure from the original paper.
  It would be great to see how we compare with other tests and plots discussed in there.
  This is a good first issue to get used to the library and coding in Julia.

- The CloudMicrophysics.jl package should be tested against a high-resolution model.
  We have chosen [PySDM](https://github.com/atmos-cloud-sim-uj/PySDM)
  as our high-resolution benchmark.
  PySDM is a package for simulating the dynamics of population of particles
  and is based on the [Super-Droplet algorithm](https://rmets.onlinelibrary.wiley.com/doi/abs/10.1002/qj.441).
  Possible tasks in this project would include testing the aerosol activation parameterization
  against PySDM in an adiabatic parcel setup, or testing the 1-moment
  microphysics parameterization against PySDM in a 1-dimensional or
  2-dimensional prescribed flow setup.
  This could be extended further into a calibration exercise using the
  [EnsembleKalmanProcesses.jl](https://github.com/CliMA/EnsembleKalmanProcesses.jl) package.

> 

Reviewer 2: ```suggestion
An example of this calibration pipeline can be seen in the documentation [here](https://clima.github.io/EnsembleKalmanProcesses.jl/dev/examples/Cloudy_example/) with an example of calibrating parameters from [Cloudy.jl](https://github.com/CliMA/Cloudy.jl).
```

- Adding a 2-moment microphysics scheme for warm rain formation to the CloudMicrophysics.jl package.
  This is a priority for us and the next development goal.
  Contributions in coding, testing, (re)deriving the parameterization equations
  and discussing microphysics parameterization assumptions are welcome!

- Adding the [P3 scheme](https://journals.ametsoc.org/view/journals/atsc/72/1/jas-d-14-0065.1.xml)
  for ice phase microphysics.

>   This is a next development goal after we implement the 2-moment scheme.

Reviewer 2: ```suggestion
  This is a next development goal after we implement the 2-moment scheme - and also a big task.
```

> - Adding an aerosol model and coupling it with the aerosol activation

Reviewer: Just a comment. I don't know whether we should have the aerosol model in `CloudMicrophysics`? But we can leave it here for now.

Author: Ha! Yeah. We might want to have it as separate repo, I agree. But can keep it in the tasks list here, for now.

[bors-img]: https://bors.tech/images/badge_small.svg
[bors-url]: https://app.bors.tech/repositories/35474

## Installation and running instructions

CloudMicrophysics.jl is a Julia registered package.
It depends on a couple of standard Julia packages as well as
  the [Thermodynamics.jl](https://github.com/CliMA/Thermodynamics.jl) and
  [CLIMAParameters.jl](https://github.com/CliMA/CLIMAParameters.jl)
  (two Julia packages developed at [CliMA](https://github.com/CliMA)).

> See the [Project.toml](https://github.com/CliMA/CloudMicrophysics.jl/blob/main/Project.toml)
>   for details.

Reviewer 3: ```suggestion
See the [Project.toml](https://github.com/CliMA/CloudMicrophysics.jl/blob/main/Project.toml)
  for a full list of CloudMicrophysics.jl's dependencies.
```

When using the CloudMicrophysics.jl inside your own project,
  the easiest way to obtain the latest stable version of the package
  and it's dependencies is to use the Julia built-in package manager

>   (accessed by pressing `]` in the Julia command prompt):

Reviewer 3: ```suggestion
  (accessed by pressing `]` in the Julia REPL):
```
